go/tasks/plugins/k8s/kfoperators/common/common_operator.go

```diff
@@ -120,6 +157,47 @@ func GetLogs(taskType string, name string, namespace string,
 	return taskLogs, nil
 }
 
+func GetMPILogs(name string, namespace string,
+	workersCount int32, launcherReplicasCount int32) ([]*core.TaskLog, error) {
+	taskLogs := make([]*core.TaskLog, 0, 10)
+
+	logPlugin, err := logs.InitializeLogPlugins(logs.GetLogConfig())
+
+	if err != nil {
+		return nil, err
+	}
+
+	if logPlugin == nil {
+		return nil, nil
+	}
+
+	// get all workers log
+	for workerIndex := int32(0); workerIndex < workersCount; workerIndex++ {
+		workerLog, err := logPlugin.GetTaskLogs(tasklog.Input{
+			PodName:   name + fmt.Sprintf("-worker-%d", workerIndex),
+			Namespace: namespace,
+		})
+		if err != nil {
+			return nil, err
+		}
+		taskLogs = append(taskLogs, workerLog.TaskLogs...)
+	}
+	// get all worker servers logs
+	for psReplicaIndex := int32(0); psReplicaIndex < launcherReplicasCount; psReplicaIndex++ {
+		// TODO: Add launcher pod name, {POD_NAME}-launcher-HASH
+		psReplicaLog, err := logPlugin.GetTaskLogs(tasklog.Input{
```

Comment:
@EngHabu @kumare3 I have one question. MPI operator creates a pod with a name template like `{POD_NAME}-launcher-HASH` and when we create a log we need the exact name of the pod. Do you have any suggestion to resolve it.